package.json

```diff
@@ -8,7 +8,7 @@
     "doc": "jsdoc -t node_modules/docdash --readme JSDOC.md -c jsdoc-config.json",
     "doclint": "npm run doc -- -t templates/silent",
     "test": "npm run lint -- --max-warnings=0 && npm run build && npm run test-examples && npm run test-unit",
-    "test-unit": "cross-env BABEL_DISABLE_CACHE=1 mocha --require babel-core/register test/*unit_test.js",
+    "test-unit": "cross-env BABEL_DISABLE_CACHE=1 mocha --require babel-core/register test/unit",
```

Comment:
Would you mind changing this as well to `unit-test` or `unit-tests`?

Reply:
I guess why not but as it is a subtask of `test`, it makes sense to keep the `test-` prefix.